src/runtime/node/perf_hooks/index.ts

```diff
@@ -0,0 +1,41 @@
+import type perf_hooks from "node:perf_hooks";
+import type { IntervalHistogram } from "node:perf_hooks";
+import * as performanceApis from "../../web/performance/index";
+import { constants } from "./constants";
+
+export { constants } from "./constants";
+export * from "../../web/performance";
+
+const mockRecordableHistogram = () =>
+  Object.create({
+    min: 9_223_372_036_854_776_000,
+    max: 0,
+    mean: null,
+    exceeds: 0,
+    stddev: null,
+    count: 0,
+    percentile: {
+      100: 0,
+    },
+  });
+
+function mockIntervalHistogram(): IntervalHistogram {
+  return Object.create({
+    ...mockRecordableHistogram(),
+    enable: () => true,
+    disable: () => true,
+  });
+}
+
+export const createHistogram: typeof perf_hooks.createHistogram = () =>
+  mockRecordableHistogram();
+export const monitorEventLoopDelay: typeof perf_hooks.monitorEventLoopDelay =
+  () => mockIntervalHistogram();
+
+// PerformanceNodeTiming is included in the types but doesn't exist in the runtime
+export default <Omit<typeof perf_hooks, "PerformanceNodeTiming">>{
```

Comment:
Checking with Node 20.12.2 REPL, i get a named export `performance` with `nodeTiming` instance:

<img width="353" alt="image" src="https://github.com/unjs/unenv/assets/5158436/c96ee797-a8d7-4d6f-9511-3905a2e27c5b">

Reply:
well that should be fine considering `performance` is a deep mock now!